lib/linguist/languages.yml

```diff
@@ -8798,3 +8798,15 @@ xBase:
   tm_scope: source.harbour
   ace_mode: text
   language_id: 421
+Xmake:
+  type: programming
+  color: "#22a079"
+  extensions:
+  - ".xmake"
+  - ".xmake.lua"
```

Comment:
These two extensions do not appear to have much usage on GitHub and will stop this PR from being merged. I recommend removing them.

Reply:
done